stackdriver/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+### 1.1.1 / 2019-06-11
```

Comment:
This version is wrong.

Reply:
This seems like it might be a bug in releasetool. I'll try to figure it out and replace this PR.